### PR TITLE
Update ConvertService for external usage 👼

### PR DESCRIPTION
**Pull request by vdemeester**
Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: cli/compose/convert/service.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func Services(
 	return result, nil
 }
 
-func convertService(
+func ConvertService(
```
Comment (dnephin) [inline cli/compose/convert/service.go]: I guess this might complain about a missing comment/godoc string?

Comment (vdemeester) [inline cli/compose/convert/service.go]: ah dang it you're right 😅

**Comment by thaJeztah** [comment]
Hm, looks like golint doesn't like it;

```
cli/compose/convert/service.go:55:6:warning: func name will be used as convert.ConvertService by other packages, and that stutters; consider calling this Service (golint)
Exited with code 1
```

**Comment by thaJeztah** [comment]
This one isn't in 17.06 - moving milestone

**Pull request merged**